### PR TITLE
Installation edit

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
@Blackbaud-AlexKingman Updated the package name

@Blackbaud-ChristiSchneider If you could confirm the step I added is correct, that would be awesome :)

Addresses https://github.com/blackbaud/skyux2-docs/issues/407 and https://github.com/blackbaud/skyux2-docs/issues/415

### CODE REVIEW

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 1]
```diff
@@ -76,6 +76,11 @@
         In the prompt that appears, select <strong>Allow</strong> to open a settings page.
```
Comment (Blackbaud-ChristiSchneider) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: This no longer opens the settings page, instead, it shows an alert that says "Profile Downloaded. Review the profile in Settings app if you want to install it.", and the only option is "Close".

So this line could say "... select <strong>Allow</strong> and then <strong>Close</strong>" and then I think it's good!

**Comment by Blackbaud-ChristiSchneider** [comment]
Looks good!

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 2]
```diff
@@ -73,7 +73,12 @@
     </li>
     <li>
       <p>
-        In the prompt that appears, select <strong>Allow</strong> to open a settings page.
+        In the prompt that appears, select <strong>Allow</strong> and then <strong>Close </strong>.
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: What do you think of adding "to download the profile" after "Allow" and adding "select" before "Close"? No big deal either way; I just thought that might add some context about what's happening between the selections.

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 3]
```diff
@@ -73,7 +73,12 @@
     </li>
     <li>
       <p>
-        In the prompt that appears, select <strong>Allow</strong> to open a settings page.
+        In the prompt that appears, select <strong>Allow</strong> and then <strong>Close</strong>.
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: What do you think of adding "to download the profile" after "Allow" and adding "select" before "Close"? No big deal either way; I just thought that might add some context about what's happening between the selections.

**Pull request merged**